lib/TileServer.js

```diff
@@ -230,7 +230,7 @@ TileServer.prototype.serve = function(req, http, callback) {
 		},
 		function finalizeResult(callback) {
 			if (!result.headers) result.headers = {};
-			result.headers['Content-Length'] = result.buffer ? result.buffer.length : 0;
+			result.headers['Content-Length'] = result.buffer && result.status !== 204 ? result.buffer.length : 0;
```

Comment:
In these cases won't the buffer be empty, which would lead to a Content-Length of 0 already? If we're not wanting to send the header at all, wouldn't it need to be something more like this?
```js
if (result.status !== 204) {
	result.headers['Content-Length'] = result.buffer.length;
}
```